canonical-web-and-design/anbox-cloud.io
# Add secondary nav and links to anbox-cloud

## Done

- Add secondary nav and links to anbox-cloud homepage
- **NOTE**: Learn more about virtual devices link from homepage links to this [copy doc](https://docs.google.com/document/d/18SJoqttKOsDZ0zLO2eqcH6rot7T69TsNzy43nQbDgJs/edit#heading=h.sh0me35kv6j0) - but I can't see this issue tracked anywhere, and this page doesn't exist? Will create an issue to add it when we firgure out what's going on

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Check the secondary nav works as expected
- Check the links against the [copydoc](https://docs.google.com/document/d/1SfpOBeDJUZj2ZYq4V8lHOUbQF9Yzo7mp5NajJDVwFMI/edit#) all there except the `Learn more about virtual devices`


## Issue / Card

Fixes #167 

## Screenshots

![Screenshot 2021-06-04 at 14 48 01](https://user-images.githubusercontent.com/58959073/120811444-e2979e00-c543-11eb-8c99-a02635ccfd70.png)
